api/room/schema.py

```diff
@@ -103,7 +105,6 @@ class Arguments:
         room_type = graphene.String()
         capacity = graphene.Int(required=True)
         image_url = graphene.String()
-        location_id = graphene.Int()
         floor_id = graphene.Int(required=True)
```

Comment:
since we are using tags, I don't think we need floors or we should include floor_id and office_id in the mutation, that's why when I run the mutation I get the following error.
<img width="1190" alt="Screenshot 2019-03-11 at 10 15 36" src="https://user-images.githubusercontent.com/23398223/54106266-a87fd600-43e6-11e9-9125-f9b7ea0019c1.png">

Reply:
True @michael-basweti, let me update the mutation.

Reply:
@michael-basweti , I don't think there is a need to remove the floors and offices because this PR was created to be merged into `develop`. The changes would be made to when it is to be merged into `v2`